en/flight_modes_rover/ackermann.md

```diff
@@ -18,16 +18,16 @@ Manual modes require stick inputs from the user to drive the vehicle.
 
 The sticks provide the same "high level" control effects over direction and rate of movement in all manual modes:
 
-- `Left stick up/down`: Drive the rover forwards/backwards (controlling speed)
+- `Left stick up/down`: Drive the rover forwards/backwards (controlling forward speed)
```

Comment:
So it doesn't control backward speed?

Reply:
No, it does control backwards too. Reworded it in this small [PR](https://github.com/PX4/PX4-user_guide/pull/3530).